### PR TITLE
Add Neon Border Button

### PR DESCRIPTION
**Pull request by RakshitaTantry28**
Issue:

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ $(document).ready(function () {
           label: "Gradient",
           classes: "gradient-btn",
         },
-       
+
         {
```
Comment (HADES-01) [inline assets/js/buttons-examples.js:63]: @RakshitaTantry28 You have to add your button to the buttons array for it to displayed on the website.

**Review by HADES-01 — CHANGES_REQUESTED** [review]
@RakshitaTantry28 And you can you please also add support for different colors as well. For example look [here](https://github.com/sButtons/sbuttons/blob/master/src/components/animated/_neon.less).

File: assets/js/buttons-examples.js (hunk 1 above)
Comment (RakshitaTantry28) [reply assets/js/buttons-examples.js:63]: Okay sure will do the changes.
Could you @HADES-01 please let me know if i have to pull request again after doing changes or it is changed throughout after pushing the code to the branch ?

Comment (HADES-01) [reply assets/js/buttons-examples.js:63]: @RakshitaTantry28 No you don't need to make a new PR for this.

Comment (RakshitaTantry28) [reply assets/js/buttons-examples.js:63]: @HADES-01 then how the changes i have made would get reflected?

File: assets/js/buttons-examples.js [hunk 2]
```diff
@@ -59,7 +59,13 @@ $(document).ready(function () {
...
           classes: "gradient-btn",
         },
-       
+        {
+          type: "neon-border",
+          label: "Neon Border",
+          text: "BUTTON",
+          classes: "neon-border-btn",
+          shouldHaveRoundedType: false,
+        },
```
Comment (HADES-01) [inline assets/js/buttons-examples.js:68]: @RakshitaTantry28 Great Work! Just two more things, Can you put the this object in alphabetical order and change the `BUTTON` to `Button`.

Comment (RakshitaTantry28) [reply assets/js/buttons-examples.js]: done!!

**Review by HADES-01 — APPROVED** [review]
@RakshitaTantry28 Great, Thank you.

File: src/components/styles/_neon-border.less [hunk 3]
```diff
@@ -0,0 +1,88 @@
+.neon-border-btn {
+  .neon(@col) {
+    color: @black;
+    background-color: @col;
+    box-shadow: inset 0 0 1em @col, 0 0 1em @col;
```
Comment (shahednasser) [inline src/components/styles/_neon-border.less:5]: Change this to `inset 0 0 6px @color, 0 0 6px @color`

File: src/components/styles/_neon-border.less [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    background: none;
+    border-radius: 0;
+    padding: 1.2em 5em;
```
Comment (shahednasser) [inline src/components/styles/_neon-border.less:8]: Remove this line

File: src/components/styles/_neon-border.less [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    letter-spacing: 0.35em;
```
Comment (shahednasser) [inline src/components/styles/_neon-border.less:9]: Remove this line

File: src/components/styles/_neon-border.less [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    font-size: 0.7em;
```
Comment (shahednasser) [inline src/components/styles/_neon-border.less:10]: Remove this line

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hello @RakshitaTantry28 thank you for your work! I need you to make a few changes below. Once you're done please remember to run `npm run compile` and commit your changes.

File: src/components/styles/_neon-border.less (hunk 6 above)
Comment (RakshitaTantry28) [reply src/components/styles/_neon-border.less]: @shahednasser i have done the respective changes, do let me know if any other?

File: src/components/styles/_neon-border.less [hunk 7]
```diff
@@ -0,0 +1,83 @@
+.neon-border-btn {
+  .neon(@col) {
+    color: @black;
+    background-color: @col;
+    background: transparent;
+    box-shadow: 0 0 6px @col, 0 0 6px @col;
```
Comment (shahednasser) [inline src/components/styles/_neon-border.less:6]: You forgot the inset here in the first box-shadow

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: src/components/styles/_neon-border.less (hunk 7 above)
Comment (RakshitaTantry28) [reply src/components/styles/_neon-border.less]: @shahednasser hey! I am sorry for so many errors, I hope i have fixed all the errors now, have a look and let me know!

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @RakshitaTantry28 for code

**Pull request merged**